### DISCUSSION TITLE
For applications going into production better to use quasar CLI than Vite?

### DISCUSSION THREAD

**Discussion by ewertonrfd** [quasarframework/quasar#13946] [General - Components / Directives / etc] [answered]
For applications going into production better to use quasar CLI than Vite?

Or the Vite I could already use for production
In the documentation I saw an alert about it's not 100% yet, it's still valid

Thank you in advance for your help

**Comment by TobyMosque** [comment, marked as answer]
Quasar CLI and Vite are like oranges and apples. 

Maybe you want to know if u would use Quasar CLI with Webpack or with Vite.
Or do u're trying to choice between Quasar CLI with Vite and the Vite CLI?

**Reply by ewertonrfd** [reply]
I understand now,
The first option would be doubt actually:

**Quasar CLI with Webpack or with Vite**

**Reply by TobyMosque** [reply]
IMHO: I didn't see a diference between both (regarding who runs better after the build).

so, If I'm not using yarn workspaces, i would go with vite (a better DX experience). (common case)
if I'm using yarn workspaces and I'd workspaces with app extensions, I would go with Webpack (I got some bugs while using Vite) (edge case).

**Reply by ewertonrfd** [reply]
IMHO 😅
Thank you so much for explaining

Eu poderia te responder em português até, sou de MG 😁
